addons/purchase_stock/tests/test_product_template.py

```diff
@@ -0,0 +1,30 @@
+from odoo.tests.common import TransactionCase
+
+
+class TestProductTemplate(TransactionCase):
+  def setUp(self):
```

Comment:
IMHO, there is no real need to define these in `setUp` since they are specific to your test case.

Reply:
I moved them directly in the test